android/app/src/main/java/org/fossasia/openevent/core/main/MainActivity.java

```diff
@@ -849,7 +847,7 @@ public void downloadFromAssets() {
     }
 
     public void readJsonAsset(final String name) {
-        CommonTaskLoop.getInstance().post(new Runnable() {
+        CommonTaskLoop.getInstance().delayPost(new Runnable() {
```

Comment:
@iamareebjamal  this works, not sure if its a good way (adding delay), is there any better possible solution for this?

Reply:
Use RxJava instead

Reply:
This is not the correct way. Check out why the dialog box is not shown in the first place